.github/workflows/dart.yml

```diff
@@ -34,35 +34,35 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
         sdk: [stable, beta]
         exclude:
+          - os: windows-latest
+            sdk: beta
           - os: macos-latest
             sdk: beta
     steps:
       - uses: dart-lang/setup-dart@6a218f2413a3e78e9087f638a238f6b40893203d # pin@v1
         with:
           sdk: ${{ matrix.sdk }}
       - uses: actions/checkout@v3
-      # coverage with 'chrome' platform hangs the build
+
       - name: Test (VM and browser)
```

Comment:
Was this not working with macOS before?

Reply:
yes, that's what the comment says:

> coverage with 'chrome' platform hangs the build

For `macOS` specifically.